app/workers/start_of_cycle_notification_worker.rb

```diff
@@ -9,7 +9,7 @@ def perform(service = 'find')
 
     providers_scope.limit(fetch_limit).each do |provider|
       provider.provider_users.each do |provider_user|
-        unless ChaserSent.exists?(chased: provider_user, chaser_type: service_open_email)
+        if !ChaserSent.where('created_at > ?', CycleTimetable.send("#{service}_opens")).exists?(chased: provider_user, chaser_type: service_open_email)
```

Comment:
This is better than the naive date, nice one

Reply:
I think we can improve this with a scope actually.

```ruby
scope :since_service_opened, ->(service) { where('created_at > ?', CycleTimetable.send("#{service}_opens")) }
```

```ruby
ChaserSent.since_service_opened(service).exists?(chased: provider_user, chaser_type: service_open_email)
```

WDYT?